components/Badge/MarketBadge.tsx

```diff
@@ -1,71 +1,59 @@
 import { CurrencyKey } from 'constants/currency';
+import useIsMarketTransitioning from 'hooks/useIsMarketTransitioing';
 import useMarketClosed from 'hooks/useMarketClosed';
-import React, { FC, useEffect, useState } from 'react';
+import React, { FC } from 'react';
 import { useTranslation } from 'react-i18next';
-import useMarketHoursTimer from 'sections/exchange/hooks/useMarketHoursTimer';
 import styled from 'styled-components';
 import theme from 'styles/theme';
 import { marketIsOpen, marketNextOpen, marketNextTransition } from 'utils/marketHours';
+
 import Badge from './Badge';
 
 type MarketBadgeProps = {
 	currencyKey: CurrencyKey | null;
-	description?: 'long' | 'short';
 };
 
-type TimerBadgeProps = {
-	timer: string | null;
+type TransitionBadgeProps = {
 	isOpen: boolean;
-	description: 'long' | 'short';
 };
 
-export const TimerBadge: FC<TimerBadgeProps> = ({ isOpen, description, timer }) => {
+export const TransitionBadge: FC<TransitionBadgeProps> = ({ isOpen }) => {
 	const { t } = useTranslation();
 
 	return (
 		<StyledBadge
 			background={isOpen ? theme.colors.common.primaryRed : theme.colors.common.primaryGreen}
```

Comment:
I think these colors are confusing.

When a market is OPEN but CLOSING SOON we have a red badge. Red indicates to me that something is closed, or that I can't interact.

When a market is CLOSED but OPENING SOON we have a green badge. Green indicates to me that something is open and I can interact, but the market is closed.

I'd prefer if the "Closing soon" badge was yellow, and the "opening soon" badge is red.